css-transitions/styles.css

```diff
@@ -0,0 +1,54 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245, 245, 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+  transition: 0.5s;
```

Comment:
It's generally better to specify which property/properties you're transitioning over:

---

![image](https://user-images.githubusercontent.com/16143771/146244248-b8e01cc0-2c2f-42be-8f38-b51b31033846.png)